builds/acr-cleanup/acr-registry-cleanup.yaml

```diff
@@ -39,6 +40,10 @@ parameters:
         olderThan: "14d"
         tagFilter: "^pr-.*"
         keep: 1
+      - repoRegex: '^labs/.*'
+        olderThan: "14d"
+        tagFilter: ".*"
+        keep: 1
```

Comment:
another value is `0` so that we can claim more storage spaces, however I am not sure if it is safe to do so